### PR TITLE
Added jython 2.5.3

### PR DESCRIPTION
**Pull request by seletz**
Added new formula for jython version 2.5.3.

### CODE REVIEW

**Comment by seletz** [comment]
Ok, seems like I need to do some RTFM.

File: jython253.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Jython < Formula
```
Comment (DomT4) [inline jython253.rb]: The `class` name here needs to match the formula name, which is the error you're getting.

**Comment by DomT4** [comment]
Couple nits to chase up on:

```
==> audit problems
homebrew/versions/jython253:
 * A `test do` test block should be added
 * Description is too long. "name: desc" should be less than 80 characters (currently 89).
```

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/97f9545144fc07637ba268d1bb6c60e6d675b11f. Thank you for your contribution to Versions, we appreciate it! :smiley_cat:.

Just a nit, but new formulae commit messages should be styled `jython253 2.5.3 (new formula)` or similar, Thanks!

**Pull request closed without merging**